src/global/interfaces/accessibility-properties.ts

```diff
@@ -11,3 +11,23 @@ export interface AccessibilityProperties {
   /** This will be forwarded as aria-labelledby to the relevant nested element. */
   accessibilityLabelledby: string | undefined;
 }
+
+/**
+ * Lists all attributes for the AccessibilityProperties interface
+ * @param accessibilityProps accessibility props
+ */
+export function getAccessibilityAttributeList(
+  accessibilityProps?: AccessibilityProperties | null
+): Record<string, string> {
+  return Object.assign({
```

Comment:
With the current usage, Object.assign will do nothing.
```suggestion
  return Object.assign({}, {
```